internal/format/format.go

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2021 Canonical Ltd
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0'
+ */
+
+package format
```

Comment:
This package and code is making the actual REST calls, correct? If so, `format` doesn't seem the best name. I was expecting just utilities to format data. Also not using the clients in go-mod-core-contracts instead of the direct REST calls?

Reply:
good point. I created this package originally with the view in mind that I might need some code to format the data as I implement the methods. However, as you pointed out, it should be using the go-mod-core-contracts implementation instead anyway. I'm replacing this now with a different implementation using go-mod-core-contracts and tabular output and will check that in later today.